<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshGeneratorWidget</class>
 <widget class="QWidget" name="MeshGeneratorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>872</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Mesh Generator</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QDockWidget" name="dockWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>353</width>
       <height>197</height>
      </size>
     </property>
     <property name="features">
      <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
     </property>
     <property name="allowedAreas">
      <set>Qt::AllDockWidgetAreas</set>
     </property>
     <property name="windowTitle">
      <string>Control Panel</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-295</y>
            <width>316</width>
            <height>1070</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QFrame" name="identifier_frame">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QLabel" name="identifier_label">
                  <property name="text">
                   <string>Identifier</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="Line" name="line">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="meshType_frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QFormLayout" name="formLayout_3">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <property name="topMargin">
                 <number>3</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="meshType_label">
                  <property name="text">
                   <string>Mesh type:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="meshType_comboBox"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="meshTypeOptions_frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_9"/>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="modifyOptions_frame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="deleteElementRanges_label">
                  <property name="text">
                   <string>Delete element ID ranges (e.g. 1,2-5,13):</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="deleteElementsRanges_lineEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="scale_label">
                  <property name="text">
                   <string>Scale x*y*z:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="scale_lineEdit"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="displayOptions_groupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Display options:</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QCheckBox" name="displayAxes_checkBox">
                  <property name="text">
                   <string>Axes</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="displayLines_checkBox">
                  <property name="text">
                   <string>Lines</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="displaySurfaces_frame">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QCheckBox" name="displaySurfaces_checkBox">
                     <property name="text">
                      <string>Surfaces</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="displaySurfacesExterior_checkBox">
                     <property name="text">
                      <string>Exterior</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="displaySurfacesTranslucent_checkBox">
                     <property name="text">
                      <string>Transluc.</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="displaySurfacesWireframe_checkBox">
                     <property name="text">
                      <string>Wireframe</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="displayElementNumbers_checkBox">
                  <property name="text">
                   <string>Element numbers</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="displayNodeNumbers_checkBox">
                  <property name="text">
                   <string>Node numbers</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="displayNodeDerivatives_checkBox">
                  <property name="text">
                   <string>Node derivatives</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="displayXiAxes_checkBox">
                  <property name="text">
                   <string>Xi axes</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="displayImagePlane_checkBox">
                  <property name="text">
                   <string>Image plane</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="displayFiducialMarkers_checkBox">
                  <property name="text">
                   <string>Fiducial markers</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="time_groupBox">
               <property name="title">
                <string>Time:</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="1" column="1">
                 <widget class="QPushButton" name="timePlayStop_pushButton">
                  <property name="text">
                   <string>Play</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="timeValue_label">
                  <property name="text">
                   <string>Time value:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="timeValue_doubleSpinBox">
                  <property name="maximum">
                   <double>12000.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QCheckBox" name="timeLoop_checkBox">
                  <property name="text">
                   <string>Loop</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="video_groupBox">
               <property name="title">
                <string>Video:</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="frameIndex_label">
                  <property name="text">
                   <string>Frame index:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="framesPerSecond_spinBox">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>25</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="framesPerSecond_label">
                  <property name="text">
                   <string>Frames per second:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="frameIndex_spinBox">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>10000</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QFrame" name="numFrames_frame">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <property name="margin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="numFrames_label">
                     <property name="text">
                      <string># frames:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="numFramesValue_label">
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="alignment_groupBox">
               <property name="title">
                <string>Alignment:</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="1" colspan="2">
                 <widget class="QComboBox" name="activeModel_comboBox">
                  <item>
                   <property name="text">
                    <string>Image Plane</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Generated Mesh</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="activeModel_label">
                  <property name="text">
                   <string>Active Model:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="toImage_pushButton">
                  <property name="text">
                   <string>To Image</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="toImage_label">
                  <property name="text">
                   <string>Align View:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QCheckBox" name="fixImagePlane_checkBox">
                  <property name="text">
                   <string>Fix image plane</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="fiducialMarkers_groupBox">
               <property name="title">
                <string>Fiducial markers:</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="1">
                 <widget class="QComboBox" name="fiducialMarker_comboBox"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="fiducialMarkerLabels_label">
                  <property name="text">
                   <string>Labels:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="fiducialMarkerTransform_label">
                  <property name="text">
                   <string>Transform:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="fiducialMarkerTransform_pushButton">
                  <property name="text">
                   <string>To Scaffold</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="blackfynn_groupBox">
               <property name="title">
                <string>Blackfynn:</string>
               </property>
               <layout class="QFormLayout" name="formLayout">
                <item row="2" column="0">
                 <widget class="QLabel" name="blackfynnDataset_label">
                  <property name="text">
                   <string>Sataset name:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="blackfynnTimeSeries_label">
                  <property name="text">
                   <string>Time series name:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="blackfynnDataset_lineEdit">
                  <property name="placeholderText">
                   <string>name</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="blackfynnTimeSeries_lineEdit">
                  <property name="placeholderText">
                   <string>name</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" colspan="2">
                 <widget class="QCheckBox" name="blackfynnShowEEG_checkBox">
                  <property name="text">
                   <string>Show EEG data in animation</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0" colspan="2">
                 <widget class="QGroupBox" name="blackfynnCredentials_groupBox">
                  <property name="title">
                   <string>Credentials:</string>
                  </property>
                  <layout class="QFormLayout" name="formLayout_2">
                   <item row="2" column="0">
                    <widget class="QPushButton" name="blackfynnCredentialsLogin_pushButton">
                     <property name="text">
                      <string>Login</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0" colspan="2">
                    <widget class="QLineEdit" name="blackfynnCredentialsAPISecret_lineEdit">
                     <property name="placeholderText">
                      <string>API Secret</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="0" colspan="2">
                    <widget class="QLineEdit" name="blackfynnCredentialsAPIKey_lineEdit">
                     <property name="placeholderText">
                      <string>API Key</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <widget class="QPushButton" name="viewAll_button">
            <property name="text">
             <string>View All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="done_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Done</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="AlignmentSceneviewerWidget" name="sceneviewer_widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>4</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AlignmentSceneviewerWidget</class>
   <extends>QWidget</extends>
   <header>opencmiss/zincwidgets/alignmentsceneviewerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
